rosdep/python.yaml

```
@@ -7488,6 +7488,19 @@ python3-multimethod-pip:
   ubuntu:
     pip:
       packages: [multimethod]
+python3-munkres:
+  debian:
+    bullseye: [python3-munkres]
+    buster: [python3-munkres]
+  fedora: [python3-munkres]
+  opensuse: [python3-munkres]
+  rhel:
+    '*': [python3-munkres]
+    '7': null
+  ubuntu:
+    bionic: [python3-munkres]
+    focal: [python3-munkres]
+    jammy: [python3-munkres]
```

The distro names can be elided since all currently supported Ubuntu distributions package this software.

```suggestion
  ubuntu: [python3-munkres]
```